packages/malloy/src/lang/ast/ast-types.ts

```diff
@@ -100,8 +101,8 @@ export class FT {
     const strings = types.map((type) => {
       if (type) {
         let inspected: string = type.dataType;
-        if (type.aggregate) {
-          inspected = `aggregate ${inspected}`;
+        if (type.expressionType != "scalar") {
+          inspected = `expressionType ${type.expressionType}`;
```

Comment:
OK, I think this is wrong.

for an aggregate timestamp, the old code would print `aggregate timestamp`
this code will print `expressionType aggregate`